schemas/context/search.schema.json

```diff
@@ -41,5 +41,5 @@
       "$ref": "#/definitions/search"
     }
   ],
-  "meta:status": "experimental"
+  "meta:status": "stabilizing"
```

Comment:
#99 is still unresolved 👎

Reply:
I think we can drop this one for the 0.9 milestone, let's plan on fixing for 1.0.